proposals/NNNN-modify-the-range-of-APT's-maxDuration.md

```diff
@@ -0,0 +1,79 @@
+# Modify the range of APT's maxDuration
+
+* Proposal: [SDL-NNNN](NNNN-modify-the-range-of-APT's-maxDuration.md)
+* Author: [zhouxin627](https://github.com/zhouxin627)

```

Comment:
Please include your name here.

Reply:
@theresalech Thanks, fixed in [ed1ca91](https://github.com/smartdevicelink/sdl_evolution/pull/930/commits/ed1ca91ae37842a0403ae2b398be5a7cdd045d52?short_path=5b49f0e#diff-5b49f0ee7a005740678492c125f406ce)